azalea/examples/nearest_entity.rs

```diff
@@ -0,0 +1,73 @@
+use azalea::nearest_entity::EntityFinder;
+use azalea::ClientBuilder;
+use azalea::{Bot, LookAtEvent};
+use azalea_client::Account;
+use azalea_entity::metadata::{Item, ItemItem};
+use azalea_entity::{EyeHeight, Position};
+use bevy_app::{FixedUpdate, Plugin};
+use bevy_ecs::{
+    prelude::{Entity, EventWriter},
+    query::With,
+    system::Query,
+};
+
+#[tokio::main]
+async fn main() {
+    let account = Account::offline("bot");
+
+    ClientBuilder::new()
+        .add_plugins(LookAtStuffPlugin)
+        .start(account, "localhost:34071")
+        .await
+        .unwrap();
+}
+
+pub struct LookAtStuffPlugin;
+impl Plugin for LookAtStuffPlugin {
+    fn build(&self, app: &mut bevy_app::App) {
+        app.add_systems(FixedUpdate, (look_at_everything, log_nearby_item_drops));
+    }
+}
+
+fn look_at_everything(
+    bots: Query<Entity, With<Bot>>,
```

Comment:
Usually I go with `With<Local>`  instead of `With<Bot>` since `Bot` isn't included for bots made with azalea-client. Doesn't matter that much but I figure for an example it should be consistent with the others.